src/main/java/org/opensearch/security/filter/SecurityFilter.java

```diff
@@ -386,15 +394,18 @@ private static boolean isUserAdmin(User user, final AdminDNs adminDns) {
     }
 
     private void attachSourceFieldContext(ActionRequest request) {
+
+        Boolean hasOdfeNodes = clusterInfoHolder.getHasOdfeNodes();
+        PackageBehavior packageBehavior = (hasOdfeNodes == null || hasOdfeNodes) ? PackageBehavior.REWRITE_AS_ODFE : PackageBehavior.NONE;
```

Comment:
why does the null check here validate to true?

Reply:
Its the default behavior, is cluster state is unknown then continue doing the rewrite behavior it does today.

Reply:
okay.. so default it assumes to have ODFE nodes.